glean-core/csharp/Glean/LibGleanFFI.cs

```diff
@@ -161,6 +161,47 @@ bool disabled
              string storage_name
          );
 
+        // Boolean
+
+        /// <summary>
+        /// A handle for the boolean metric type, which performs cleanup.
+        /// </summary>
+        internal sealed class BooleanMetricTypeHandle : BaseGleanHandle
+        {
+          protected override bool ReleaseHandle()
+          {
+            if (!this.IsInvalid)
+            {
+              Console.WriteLine("Freeing boolean metric type handle");
```

Comment:
Argh, I forgot to remove this in my code as well. We don't need it, can you remove it both here and [there](https://searchfox.org/glean/rev/b7c91243657df02f3ef2d579d02b27661c23e54e/glean-core/csharp/Glean/LibGleanFFI.cs#127)?